### PR TITLE
HW2-jp4013

### PR DESCRIPTION
**Pull request by jaesungpark42**
I cannot find why it won't automatically merge yet can still let me rebase yet.

### CODE REVIEW

File: projects/python/chatGPTCoverLetterGenerator.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+New Project Idea:
```
Comment (timkpaine) [inline projects/python/chatGPTCoverLetterGenerator.md:1]: include link to project repo

File: jp4013.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ Name: Jaesung Park
```
Comment (timkpaine) [inline jp4013.md:4]: this file should've ended up in `students/`

File: jp4013.md [hunk 3]
```diff
@@ -13,7 +15,15 @@ GitHub README: [link](https://github.com/jaesungpark42/jaesungpark42/blob/main/R
 -Job Application Autofiller: There are many, but each lack flexibility on different variety of job application versions, also the only open source one is really small and lacking https://github.com/lovincyrus/job-autofiller  
 -Saving multiple pages of websites to a pdf. 
 
+**LINK** 
+[chatGPTCoverLetterGenerator](./projects/python/chatGPTCoverLetterGenerator.md)
```
Comment (timkpaine) [inline jp4013.md:19]: update path after moving to `students`, e.g. `../projects/...`

File: jp4013.md [hunk 4]
```diff
@@ -13,7 +15,15 @@ GitHub README: [link](https://github.com/jaesungpark42/jaesungpark42/blob/main/R
...
+
+
 ## Project Contributions
 -Could be the job autofiller, but it is really small, I don't know if it would be an average experience of contributing to open source that this project intends to encourage.
 
 Really open to suggestions for contributions. I don’t have many tools/libraries that I use regularly that I have a problem with.
+
+**LINK**
+[jobAppAutofiller](./projects/javascript/jobAppAutofiller.md)
```
Comment (timkpaine) [inline jp4013.md:28]: update path

**Review by timkpaine — CHANGES_REQUESTED** [review]
ensure all commits squashed into 1

**Comment by jaesungpark42** [comment]
It let me squash the changes on rebase but it doesn't show up here. I'm confused.

**Pull request closed without merging**